libs/domain/cart/list-item/list-item.component.ts

```diff
@@ -0,0 +1,147 @@
+import { CartComponentMixin, CartService, PriceMode } from '@spryker-oryx/cart';
+import { resolve } from '@spryker-oryx/di';
+import { RouteType } from '@spryker-oryx/router';
+import { LinkService } from '@spryker-oryx/site';
+import { AlertType } from '@spryker-oryx/ui';
+import { I18nMixin, Size, signal } from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+import { CartListItemProperties } from './list-item.model';
+import { cartListItemStyles } from './list-item.styles';
+
+const size = Size.Md;
+
+export class CartListItemComponent
+  extends CartComponentMixin(I18nMixin(LitElement))
+  implements CartListItemProperties
+{
+  static styles = cartListItemStyles;
+
+  protected cartService = resolve(CartService);
+  protected linkService = resolve(LinkService);
+
+  @property({ type: Boolean }) open?: boolean;
+
+  protected setDefault(): void {
+    this.cartService.updateCart({
+      cartId: this.$cart()?.id,
+      isDefault: true,
+    });
+  }
+
+  protected override render(): TemplateResult | void {
+    const cart = this.$cart();
+
+    if (!cart) return;
+
+    return html`
+      <oryx-collapsible ?open=${this.open}>
+        ${this.renderHeading()} ${this.renderEntries()} ${this.renderControls()}
+      </oryx-collapsible>
+    `;
+  }
+
+  protected $cartLink = signal(this.linkService.get({ type: RouteType.Cart }));
+
+  protected renderHeading(): TemplateResult {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const cart = this.$cart()!;
+
+    return html`
+      <span slot="heading">
+        ${cart.name}
+        ${this.i18n('carts.totals.<count>-items', {
+          count: this.$totalQuantity() ?? 0,
+        })}
+      </span>
+      ${when(
+        cart.isDefault,
+        () =>
+          html`<oryx-chip slot="heading" .appearance=${AlertType.Success}>
+            ${this.i18n('default')}
+          </oryx-chip>`
+      )}
+
+      <oryx-site-price
+        slot="heading"
+        .value=${cart.totals?.priceToPay ?? 0}
+        .currency=${cart.currency}
+      ></oryx-site-price>
+
+      <span slot="heading"
+        >(
+        ${this.i18n(
+          cart.priceMode === PriceMode.GrossMode
+            ? 'cart.mode.gross'
+            : 'cart.mode.net'
+        )}
+        )</span
+      >
+    `;
+  }
+
+  protected renderControls(): TemplateResult {
+    const disabled = !this.$totalQuantity();
+
+    return html`
+      <div class="meta">
+        ${when(
```

Comment:
i'm not a big fan of using `when` directive, rather create the template outside the html (e.g. `callToActionTemplate = this.$cart()?.isDefault ? ...` and use it inside the html template. can be an inline. Also an opinionated nitpick that you can ignore ;)

Reply:
I would say that we are using `when` almost everywhere and it is a bit long, but much more readable

Reply:
must be only me then, I don't agree it's better readable